form/data_mappers.rst

```diff
@@ -150,6 +150,8 @@ method to configure the data mapper::
 
     use App\Form\DataMapper\ColorMapper;
     use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\OptionsResolver\OptionsResolver;
```

Comment:
Thanks for fixing this :). What about adding a use for the `AbstractType` too then?

Reply:
Added! Thank you :slightly_smiling_face: